sdk/openai/openai/test/public/node/whisper.spec.ts

```diff
@@ -15,41 +15,47 @@ import {
 } from "../utils/utils.js";
 import { assertAudioResult } from "../utils/asserts.js";
 import { AudioResultFormat } from "../utils/audioTypes.js";
+import { whisperModels } from "../utils/models.js";
 
-// TODO: Unskip the tests
-describe.skip("OpenAI", function () {
-  matrix([APIMatrix] as const, async function (authMethod: APIVersion) {
-    describe(`[${authMethod}] Client`, () => {
+const timeoutOption = { timeout: 2000 };
+
+describe("OpenAI", function () {
+  matrix([APIMatrix] as const, async function (apiVersion: APIVersion) {
+    describe(`[${apiVersion}] Client`, () => {
       let client: AzureOpenAI | OpenAI;
       let deployments: DeploymentInfo[] = [];
+
+      // TODO: Change to "audio" deployments once retry behavior is fixed
       beforeAll(async function () {
-        deployments = await getDeployments("audio");
+        deployments = await getDeployments("vision");
```

Comment:
Looks good but still feels weird, we probably should talk to the team to increase the rate limit in the audio resource.

Reply:
I believe this is a known issue for the audio deployment based on [this conversation in Teams](https://teams.microsoft.com/l/message/19:BcIHBYot3jta4FP7KW1c26ZfW3K_JU5X7hGlfD0V4_41@thread.tacv2/1709594008944?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=f0cde33b-5214-4d6f-8179-5bfa3a61a390&parentMessageId=1709594008944&teamName=Azure%20OpenAI%20Developer%20Experience&channelName=General&createdTime=1709594008944). I can check back to see if anything has changed since last month, but based on the last replied, it sounds like we are advised to test Whisper with this resource instead